src_c/math.c

```diff
@@ -792,8 +807,11 @@ vector_nonzero(pgVector *self)
 static PyObject *
 vector_copy(pgVector *self)
 {
-    pgVector *ret = (pgVector *)pgVector_NEW(self->dim);
+    pgVector *ret;
     Py_ssize_t i;
+
+    ret = (pgVector *)_vector_subtype_new(Py_TYPE(self), self->dim);
+
```

Comment:
```suggestion
    if (!ret) {
        return NULL;
    }
```
Not really a bug introduced by this PR but can be fixed here, this is missing error forwarding return